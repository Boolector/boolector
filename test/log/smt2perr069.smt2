(assert (not (_ bv0 2)))
