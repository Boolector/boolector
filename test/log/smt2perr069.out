log/smt2perr069.smt2:1:14: unexpected bit-vector of width 2 as argument to 'not'
